.gitignore

```diff
@@ -23,3 +23,7 @@ coverage.xml
 distribute*.gz
 venv/
 env*/
+.*
+!/.gitignore
+!/.flake8
+!/.travis.yml
```

Comment:
Not a huge fan of this pattern, especially since there is another `dot file` that already exists in the repository that is ignored by this policy: `.hgignore`.

Not that it matters much, but I prefer ignoring more specific, vs ignoring all and then whitelisting.

-0 on this change.